
#request

---

#response
uint8[] raw_stcm
